### DISCUSSION TITLE
Help with optimizing config

### DISCUSSION THREAD

**Discussion by hhdo** [swar/Swar-Chia-Plot-Manager#514] [Q&A]
Equipments:
SSD : SATA 1T *2 (Un-popular Brand) RAID0 IN WIN10
Mem : 32G (16g 3200Ghz *2)
Cpu: 5800X (8C 16T)
motherboard: MSI MOTOR WIFI
 I try to plot, but I found that the speed is very fast when 4 tasks are concurrent in the initial stage. Once 8 tasks are running in parallel at the same time, the speed will be very slow. I suspect there is something wrong with my SSD. DOes anyone has optimization methods?
![image](https://user-images.githubusercontent.com/32031637/118588793-b3f29700-b7d1-11eb-9492-f089e59f98b4.png)

Some key parameter list below: 

manager:
  check_interval: 60
  log_level: ERROR

global:
  max_concurrent: 8

jobs:
    name: main
    max_plots: 36
    farmer_public_key:
    pool_public_key:
    temporary_directory: D:\
    temporary2_directory:
    destination_directory: E:\
    size: 32
    bitfield: true
    threads: 3
    buckets: 128
    memory_buffer: 3800
    max_concurrent: 8
    max_concurrent_with_start_early: 9
    stagger_minutes: 1
    max_for_phase_1: 4
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false

Thank u

**Comment by frviana** [comment]
Set stagger_minutes:  to 60 but you should start like that. Either wait till finish or kill the process based on PID and clear the temp files on the NVMe. If you think about it, after 24hrs you should get ~24 plots in 24hrs as you will have each finishing 1hrs after each other. 24 plots a day is not bad.

**Reply by hhdo** [reply]
Maybe it has something wrong with my sata SSD, cause 4k speed is very low, which is not beyond 30

**Reply by frviana** [reply]
Nothing wrong with your SSD, it's actually doing quite well considering people are using NVMe which is usually faster. You are killing it by setting 1mim staggering

**Reply by hhdo** [reply]
ok, I try to fix it and thank you very much

**Reply by Kali123411** [reply]
If your resources are getting bogged down because of to many plots happening at one time, you can go into the resource monitor and just suspend the processes on half of them or however many, let the rest finish and then resume the processes again. That way you aren't wasting the temp plots you already have.

**Reply by hhdo** [reply]
Thank you, how to entry the resource monitor? Can this be suspended? I never knew it！

**Reply by hhdo** [reply]
> Nothing wrong with your SSD, it's actually doing quite well considering people are using NVMe which is usually faster. You are killing it by setting 1mim staggering

![image](https://user-images.githubusercontent.com/32031637/118749597-69841f80-b890-11eb-980a-08ab9520f047.png)
It seems that it still slow.😔
